### DISCUSSION TITLE
Downloading update... notification frozen

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#21026] [Q&A]
![image](https://user-images.githubusercontent.com/74088315/198942030-94d4917a-b122-4d6b-af6e-593de037d1db.png)

The game downloaded the newest version of lazer and when I restarted my game, it updated properly, but the notification seemed to be frozen as if the update wasn't working. The necessary files were downloaded and were in AppData\Local\osulazer\app-2022.1031.0 but I never got the 'Update ready. Click here to restart!' notification or any sort of progress bar even after waiting for 10 minutes.

[updater.log](https://github.com/ppy/osu/files/9898423/updater.log)
[database.log](https://github.com/ppy/osu/files/9898424/database.log)
[network.log](https://github.com/ppy/osu/files/9898425/network.log)
[runtime.log](https://github.com/ppy/osu/files/9898426/runtime.log)

**Comment by peppy** [comment]
Weirdly, your log shows that the notification text should have been updated to "Installing update...", but you never saw this?

**Reply by Flamiii** [reply]
Nope, the notification didn't update at all and was just stuck with no progress bar or anything like that.
